dsn-2018/bare_conf.tex

```diff
@@ -0,0 +1,1201 @@
+% conference papers do not typically use \thanks and this command
+% is locked out in conference mode. If really needed, such as for
+% the acknowledgment of grants, issue a \IEEEoverridecommandlockouts
+% after \documentclass
+
+% for over three affiliations, or if they all won't fit within the width
+% of the page, use this alternative format:
+%
+%\author{\IEEEauthorblockN{Michael Shell\IEEEauthorrefmark{1},
+%Homer Simpson\IEEEauthorrefmark{2},
+%James Kirk\IEEEauthorrefmark{3},
+%Montgomery Scott\IEEEauthorrefmark{3} and
+%Eldon Tyrell\IEEEauthorrefmark{4}}
+%\IEEEauthorblockA{\IEEEauthorrefmark{1}School of Electrical and Computer Engineering\\
+%Georgia Institute of Technology,
+%Atlanta, Georgia 30332--0250\\ Email: see http://www.michaelshell.org/contact.html}
+%\IEEEauthorblockA{\IEEEauthorrefmark{2}Twentieth Century Fox, Springfield, USA\\
+%Email: homer@thesimpsons.com}
+%\IEEEauthorblockA{\IEEEauthorrefmark{3}Starfleet Academy, San Francisco, California 96678-2391\\
+%Telephone: (800) 555--1212, Fax: (888) 555--1212}
+%\IEEEauthorblockA{\IEEEauthorrefmark{4}Tyrell Inc., 123 Replicant Street, Los Angeles, California 90210--4321}}
+
+
+
+
+% use for special paper notices
+%\IEEEspecialpapernotice{(Invited Paper)}
+
+
+
+
+% make the title area
+\maketitle
+
+% As a general rule, do not put math, special symbols or citations
+% in the abstract
+\begin{abstract}
+The discovery of fast collision attacks in cryptographic hash functions has
+traditionally resulted in the immediate deprecation of that hash function.
+In this paper we propose five scalable and practical metrics for evaluating
+the utility of collision classes based on boolean constraints and show that
+the published attacks by X. Wang, Y. Sasaki, P. Kasselman, H. Dobbertin, and
+M. Schl{\"a}ffer in MD4 have high utility. We expand on existing attacks by
+developing a series of techniques based on logical cryptanalysis to find
+over 35,000 collisions in MD4 based on existing collisions, through the novel
+definition of a collision neighborhood. We demonstrate new techniques for
+inductively building full collisions from reduced round variants of MD4. We
+propose these techniques as a mechanism for measuring hash trustworthiness and
+discuss potential applications to real-world systems.
+\end{abstract}
+
+
+
+
+
+
+% no keywords
+% For peer review papers, you can put extra information on the cover
+% page as needed:
+% \ifCLASSOPTIONpeerreview
+% \begin{center} \bfseries EDICS Category: 3-BBND \end{center}
+% \fi
+%
+% For peerreview papers, this IEEEtran command inserts a page break and
+% creates the second title. It will be ignored for other modes.
+\IEEEpeerreviewmaketitle
+
+\section{Introduction}
+
+Cryptographic hash functions form the core of many protocols. From file
+integrity checks to verify long term storage of data, to cache invalidation
+techniques, and use as a building block in network protocols such as Kerberos
```

Comment:
"and as as building block" [take out "use"] or change to "and **used** as a building block"